examples/text-sentiment/start_runtime.py

```diff
@@ -19,16 +19,13 @@
 # First Party
 import alog
 
+# Local
+from caikit.runtime import grpc_server
+
 sys.path.append(
     path.abspath(path.join(path.dirname(__file__), "../"))
 )  # Here we assume that `start_runtime` file is at the same level of the `text_sentiment` package
 
-# Local
-import text_sentiment
```

Comment:
We configure the library at import time, so while this looks unused, I would expect the removal here to break the example. We may want to add a `disable=unused-import` line here

Reply:
ouch thanks for catching that.